src/Pos/Explorer/Socket/Methods.hs

```diff
@@ -159,6 +159,7 @@ subscribeAddr caddr sessId = do
         sessId
         (sformat ("address "%shown) addr)
         (csAddressSubscribers . at addr . non S.empty . at sessId)
+    logDebug $ sformat ("Client #"%shown%" subscribeAddr") sessId
```

Comment:
Keh, `subscribe` function already performs logging, do we need to do it here?
And same for other changes